tests/test_values.py

```diff
@@ -392,7 +396,7 @@ def mock_database_url_caster(self, url, engine=None):
     def test_email_url_value(self):
         value = EmailURLValue()
         self.assertEqual(value.default, {})
-        with env(EMAIL_URL='smtps://user@domain.com:password@smtp.example.com:587'):
+        with env( EMAIL_URL='smtps://user@domain.com:password@smtp.example.com:587'):  # noqa: E501
```

Comment:
Props for "E501", but you are lucky that we do not run flake8 for tests (see https://github.com/jazzband/django-configurations/pull/180).
Will push a fixup for this one.